mopidy/audio/tags.py

```diff
@@ -78,7 +78,16 @@ def _extract_sample_data(sample):
     buf = sample.get_buffer()
     if not buf:
         return None
-    return buf.extract_dup(0, buf.get_size())
+    # Fix for https://github.com/mopidy/mopidy/issues/1827
```

Comment:
Only nit I have is that I would isolate this workaround in a small helper function, and provide a bit more context of why this is needed. "Python bindings before version NNN leak memory if we use extract_dub directly, this function works around this by using different APIs to achieve the same end result"?

Reply:
I did put all that info in the commit message, can reproduce here if you think that's helpful. Yeh?

Reply:
A function with the commit text as a comment sounds perfect!

Reply:
This was a good nitpick - cheers!